index.js

```diff
@@ -314,6 +342,35 @@ class MongoTenant {
     return MongoTenantModel;
   }
 
+  /**
+   * Create db connection bound to a specific tenant
+   *
+   * @param {Connection} unboundDb
+   * @param {*} tenantId
+   * @returns {Connection}
+   */
+  createBoundDb(unboundDb, tenantId) {
+    const me = this;
+    const boundDb = Object.create(unboundDb);
+    boundDb.model = (name) => {
+      const unboundModel = unboundDb.model(name);
+      const otherPluginPresent = unboundModel.schema.plugins.some(
+        plugin => (plugin.fn === mongoTenantPlugin)
```

Comment:
i think the other plugin does not need to be the same module :thinking:
in scenarios where you split your models up into separate packages we might have different mongo-tenant references.